### PR TITLE
how-to/scale: add necessary actions on the target machines

### PR DESCRIPTION
**Pull request by superlzs0476**
### What is changed, added or deleted? 

add Install the NTP service on the target machines
add Configure the SSH mutual trust and sudo rules on the Control Machine
### What is the related PR or file link(s)? 

- Reference link(s):
- This PR is to align with:
- [ ] N/A (not applicable)

**Important Notice:**

**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:

- **needs-cherry-pick-3.0**
- **needs-cherry-pick-3.1**
- **needs-cherry-pick-2.1**

### CODE REVIEW

**Comment by TomShawn** [comment]
@superlzs0476 Please fix the CI failure, thanks!

**Comment by yikeke** [comment]
麻烦和作者 @superlzs0476 确认一下这个 PR 的改动是否只适用于 v3.0。 @TomShawn

File: how-to/scale/with-ansible.md [hunk 1]
```diff
@@ -85,12 +87,37 @@ TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
...
+    username = tidb
+    ntp_server = pool.ntp.org
+    ```
 
 2. 初始化新增节点：
 
     {{< copyable "shell-regular" >}}
 
     ```bash
+    # 在中控机上配置部署机器 SSH 互信及 sudo 规则
```
Comment (TomShawn) [inline how-to/scale/with-ansible.md]: It is recommended to separate comments from commands and add the code snippet `{{< copyable "shell-regular" >}}` for each command, which improves usability.

**Comment by TomShawn** [comment]
> 麻烦和作者 @superlzs0476 确认一下这个 PR 的改动是否只适用于 v3.0。 @TomShawn

作者确认适用于 dev/v2.1/v3.0/v3.1。

**Comment by liubo0127** [comment]
LGTM

**Pull request merged**